kubemarine/core/group.py

```diff
@@ -677,7 +680,7 @@ def _await_rebooted_nodes(self, timeout=None, initial_boot_history: NodeGroupRes
                 time.sleep(delay_period - attempt_time)
 
         if left_nodes:
-            self.cluster.log.verbose("Failed to wait for boot of nodes %s." % list(left_nodes.keys()))
+            self.cluster.log.verbose("Failed to wait for boot of nodes %s. Try to increase boot_timeout parameter in node_defaults: https://github.com/Netcracker/KubeMarine/blob/main/documentation/Installation.md#node_defaults" % list(left_nodes.keys()))
```

Comment:
This is misleading when removing of offline nodes. Such check have different timeout of 30 seconds and is currently not configurable.

Reply:
Yes, this message is removed to avoid confusion.